app/e2e/incomingCall.test.ts

```diff
@@ -27,16 +27,18 @@ describe('Incoming Call', () => {
     await expect(element(by.text('Ahoy!'))).toBeVisible();
     await expect(element(by.text('test_email@twilio.com'))).toBeVisible();
 
-    await twilioClient.calls.create({
+    /**
+     * Wait for 10 seconds to let the registration settle.
+     */
+    await new Promise((resolve) => setTimeout(resolve, 10000));
+
+    const testCall = await twilioClient.calls.create({
       twiml: '<Response><Say>Ahoy, world!</Say><Pause length="5" /></Response>',
       to: `client:${clientId}`,
       from: 'detox',
     });
 
-    /**
-     * Wait for 10 seconds to let the registration settle.
-     */
-    await new Promise((resolve) => setTimeout(resolve, 10000));
+    console.log(`Call created with SID: "${testCall.sid}".`);
```

Comment:
Is this log necessary?

Reply:
Helps with debugging if the call doesn't make it through somehow. Do we want to remove it? WDYT @charliesantos

Reply:
I think it helps to know the callsid.